switchdev_utils/config-service/sample_config.yaml

```diff
@@ -8,24 +8,25 @@ pfs:
   - name: ens2f1
     numOfVfs: 8
     switchdev: True
-    vfTrust: True 
+    vfTrust: True
   - name: ens1f0
     numOfVfs: 8
     switchdev: True
     vfTrust: True
+    mtu: 9000
   - name: ens1f1
     numOfVfs: 8
     switchdev: True
     vfTrust: True
+    mtu: 9000
 linuxBonds:
   - name: bond0
     slaves: [ens2f0, ens2f1]
     bondingOptions: "mode=802.3ad miimon=100 lacp_rate=fast xmit_hash_policy=layer2+3"
-  - name: bond1
-    slaves: [ens1f0, ens1f1]
-    bondingOptions: "mode=802.3ad miimon=100 lacp_rate=fast xmit_hash_policy=layer2+3"
+    mtu: 9000
+
```

Comment:
Any reason to change the configuration from bond to pf ?

Reply:
@adrianchiris 
I wanted to cover all cases that the script is doing and especially for mtu:
- The sriov_pf could be a bond slave so only setting mtu to the bond needed, no need to set it to the slaves
- The sriov_pf could be an ovs port, so setting mtu only for it, no need to set it to the bridge
- The sriov_pf could be single interface, so setting mtu to it directly